### PR TITLE
Fix yandex organic detect

### PR DESCRIPTION
**Pull request by KuznetsovEvgeniy**

### CODE REVIEW

File: src/utmParamsFromReferrer.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { parse as parseQuery } from './queryString'
 const engines = [
   { regex: /www.google./, query: 'q', name: 'google' },
   { regex: /yandex..*\/.*\?.*text=/, query: 'text', name: 'yandex' },
+  { regex: /^https?:\/\/(www\.)?yandex\.([a-z]{2,3})\/?$/, query: 'text', name: 'yandex' },
```
Comment (MaksimKiselev) [inline src/utmParamsFromReferrer.js]: `$` on end of regex means end of string.
So, it's okay that this regex **will not** match `http://yandex.ru/search`?

Also `^` means that start of string, so only url started from `http` will match.

Comment (KuznetsovEvgeniy) [reply src/utmParamsFromReferrer.js]: yes, only .ru or .ru/ ended!

Comment (KuznetsovEvgeniy) [reply src/utmParamsFromReferrer.js]: http/https started

Comment (KirillTemnov) [reply src/utmParamsFromReferrer.js]: I think you should move this line before
```{ regex: /yandex..*\/.*\?.*text=/, query: 'text', name: 'yandex' },```

Comment (KuznetsovEvgeniy) [reply src/utmParamsFromReferrer.js]: https://monosnap.com/file/hIN99ogDJWKFjcYUUUapAM1fOoqaDb for example. More info you can see in team-segmentstream chat

**Pull request merged**